### ISSUE TITLE
White line on a11y graphic

### ISSUE THREAD

**Issue by sethjohnsonIBM** [carbon-design-system/carbon-website#520]
<!--

Thanks in advance for your feedback on the Carbon website!

In all browsers tested (Firefox, Safari, Chrome) a vertical white line appears to the left of the white circle at certain window widths—see screenshot. 

<img width="241" alt="Screen Shot 2019-10-23 at 12 42 05 PM" src="https://user-images.githubusercontent.com/32574988/67419691-c9222800-f592-11e9-94f0-d05945394b93.png">

**Comment by aagonzales** [comment]
Yup, I'm still seeing it. Looks like it has something to do with the image asset itself. 
- [ ] export image asset with transparent background and make sure there is no lingering shapes.

**Comment by aagonzales** [comment]
We've updated the accessibility section for v11 and this has been fixed
https://v11.carbondesignsystem.com/guidelines/color/overview/#accessibility
